### PR TITLE
Fix Hybrid Sample App

### PR DESCRIPTION
**Pull request by kchitalia**
@qren Please take a look when you get a chance.

### CODE REVIEW

File: Samples/Salesforce.Sample.SmartSyncExplorerHybrid/js/default.js [hunk 1]
```diff
@@ -6,6 +6,9 @@
     var activation = Windows.ApplicationModel.Activation;
     var oauth = new SalesforceJS.OAuth2();
     var smartSync = new SmartSyncJS.SmartSync();
+    var smartstore = new SmartStoreJS.SmartStore();
+    var smartsync = new SmartSyncJS.SmartSync();
```
Comment (helenren) [inline Samples/Salesforce.Sample.SmartSyncExplorerHybrid/js/default.js:9]: Is this duplicate the one above?

File: Samples/Salesforce.Sample.SmartSyncExplorerHybrid/typescript/salesforce.windows.smartstore.ts [hunk 2]
```diff
@@ -318,78 +306,82 @@ module SmartStoreJS {
...
-            var sm = this.getSmartStore(payload.isGlobalStore);
+            console.log("SmartStore.clearSoup:soupName=" + args[0]);
+            var sm = this.getSmartStore(false);
             if (!sm) {
                 errorCB("No active account");
             } else {
-                sm.clearSoup(payload.soupName);
-                successCB();
+                sm.clearSoup(args[0]);
+                successCB("Soup" + args[0] + "cleared successfully");
```
Comment (helenren) [inline Samples/Salesforce.Sample.SmartSyncExplorerHybrid/typescript/salesforce.windows.smartstore.ts]: should be "Soup " + args[0] + " is cleared successfully"

File: Samples/Salesforce.Sample.SmartSyncExplorerHybrid/typescript/salesforce.windows.smartsync.js [hunk 3]
```diff
@@ -47,22 +47,18 @@ var SmartSyncJS;
...
-            var payload = args[1];
             var syncmanager = this.getInstance();
             if (!syncmanager) {
                 fail("Error in getting instance for SmartSync");
             }
             else {
-                var syncState = syncmanager.syncDown(payload.target.asJson(), payload.soupName, null, payload.options);
+                var syncState = syncmanager.syncDown(args[0], args[1], args[2], args[3]);
                 success(syncState);
+                fail("Error in Sync Down");
```
Comment (helenren) [inline Samples/Salesforce.Sample.SmartSyncExplorerHybrid/typescript/salesforce.windows.smartsync.js]: will cal success ad fail both, or should based the real status to decide to call success or fail?

File: Samples/Salesforce.Sample.SmartSyncExplorerHybrid/typescript/salesforce.windows.smartsync.js [hunk 4]
```diff
@@ -73,21 +69,18 @@ var SmartSyncJS;
...
-                return;
             var syncmanager = this.getInstance();
             if (!syncmanager) {
                 fail("Error in getting instance for SmartSync");
             }
             else {
-                var syncState = syncmanager.syncUp(args[1], args[2], args[3], null);
+                var syncState = syncmanager.syncUp(args[0], args[1], args[2], null);
                 success(syncState);
+                fail("Error in Sync Up");
```
Comment (helenren) [inline Samples/Salesforce.Sample.SmartSyncExplorerHybrid/typescript/salesforce.windows.smartsync.js]: same comments as the above one

**Comment by kchitalia** [comment]
@qren Addressed your comments in the latest commit.

**Comment by helenren** [comment]
There is an issue on my machine that I cannot load project. So, there is no way for me to have a try. The codes looks good for me. You can go ahead merge the change. Thanks.

**Pull request merged**